lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1651,9 +1651,9 @@ fc_extras
 
     ################################################################################
     def get_attr_units(cf_var, attributes):
-        attr_units = getattr(cf_var, CF_ATTR_UNITS, cf_units._UNIT_DIMENSIONLESS)
+        attr_units = getattr(cf_var, CF_ATTR_UNITS, cf_units._UNKNOWN_UNIT_STRING)
```

Comment:
I see no need here to reference private bits of cf_units !
We should set up our own global for this, with something like `UNKNOWN_UNIT_STRING = str(cf_units(None))`

Reply:
Would it make sense to change all the other references to private bits of cf_units to something similar? i.e. cf_units._NO_UNIT_STRING.